### PR TITLE
Jakub's certification post

### PR DESCRIPTION
**Pull request by lauracowen**

### CODE REVIEW

File: posts/2023-12-15-liberty-certification.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+== Prepare for the certification exam
+
+You can prepare for the certification exam by using the free resources on the link:/[Open Liberty website], including the link:/guides/[cloud-native developer guides] and the link:/docs/[Liberty docs]. Both the guides and docs are relevant whether you are working with link:/start/[Open Liberty] or with link:https://www.ibm.com/products/websphere-hybrid-edition/liberty[IBM WebSphere Liberty] (which is built from the Open Liberty project).
+
+=== Cloud-native Java developer guides
+
+The 60 link:/guides/[cloud-native developer guides] help you to get hands-on with the technologies and tools that are available to use with Open Liberty. The guides are a great way for you to refresh your knowledge about a technology or to discover new ones. They range in complexity, from link:/guides/getting-started.html[Getting started with Open Liberty] and link:/guides/rest-intro.html[Creating a RESTful web service] to topics such as link:/guides/microprofile-metrics.html[Providing metrics from a microservice], and link:/guides/social-media-login.html[Authenticating users through social media providers].
```
Comment (dmuelle) [inline posts/2023-12-15-liberty-certification.adoc]:
```suggestion
The 60 link:/guides/[cloud-native developer guides] give you hands-on experience with the technologies and tools that you can use with Open Liberty. The guides are a great way for you to refresh your knowledge about a technology or to discover new ones. They range in complexity, from link:/guides/getting-started.html[Getting started with Open Liberty] and link:/guides/rest-intro.html[Creating a RESTful web service] to topics such as link:/guides/microprofile-metrics.html[Providing metrics from a microservice], and link:/guides/social-media-login.html[Authenticating users through social media providers].
```

File: posts/2023-12-15-liberty-certification.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+=== Liberty documentation
+
+On the link:/[Open Liberty website] you can also find the link:/docs/[Liberty docs] which describe how to use the features available with Open Liberty, including the MicroProfile and Jakarta EE specifications.
+
+image::img/blog/docs.png[Screenshot of Open Liberty docs,width=70%,align="center"]
+
+== Find out more
+
+By using the above resources, and your own knowledge and experience of writing cloud-native Java applications, you're ready to try the Liberty certification exam - good luck! Read more about the certification on the link:https://www.ibm.com/training/certification/C9004800[IBM Training portal].
```
Comment (dmuelle) [inline posts/2023-12-15-liberty-certification.adoc]:
```suggestion
By using these resources, and your own knowledge and experience of writing cloud-native Java applications, you're ready to try the Liberty certification exam - good luck! Read more about the certification on the link:https://www.ibm.com/training/certification/C9004800[IBM Training portal].
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
minor edits- mainly to remove "above"

**Review by yeekangc — APPROVED** [review]
LGTM

**Pull request merged**